pkg/generators/models/generate.go

```diff
@@ -280,6 +280,7 @@ func (g generator) writeModelToFile(ctx context.Context, model *Model, dst strin
 	log.Debug().Msg("Formatting the rendered code...")
 	content, err := format.Source(buf.Bytes())
 	if err != nil {
+		fmt.Println(string(buf.Bytes()))
```

Comment:
```suggestion
```

Reply:
I thought I forgot this, but then I thought we had a linter to catch it, so i then thought I had removed it because CI was passing